Projects/hCoreUI/Sources/Views/DetailsTableView.swift

```diff
@@ -9,31 +9,51 @@ import hGraphQL
 public struct DetailAgreementsTableView: View {
     var table: DetailAgreementsTable
 
+    var hasTitle: Bool {
+        table.title.count > 0
+    }
+
     public var body: some View {
-        ForEach(table.sections, id: \.hashValue) { section in
-            hSection(section.rows, id: \.title) { row in
-                hRow {
-                    VStack(alignment: .leading) {
-                        hText(row.title, style: .body)
-                            .foregroundColor(hLabelColor.primary)
-
-                        if let subtitle = row.subtitle {
-                            hText(subtitle, style: .subheadline)
+        VStack(alignment: .leading, spacing: 0) {
```

Comment:
Can't `hForm` be used instead of `VStack`?
Since most elements inside are `hSection` and `hRow`

Reply:
No, there is already a scrollView in the top, `hForm` would add another scrollview 😢